dimagi/pages/templates/sections/about/company_info.html

```diff
@@ -54,7 +54,7 @@ <h3>
                target="_blank">B Corp</a>, and certified
             Benefit Corporation, reflecting our commitment to making an
             impact.
-            <a href="https://cdn2.hubspot.net/hubfs/503070/2018-03-13%20-%20Dimagi%20-%20MA%20Annual%20Report%20to%20Secretary%20of%20State.pdf"
+            <a href="https://cdn2.hubspot.net/hubfs/503070/2018%20MA%20Annual%20Report%20Submission.pdf"
```

Comment:
it is cleaner to do this change in a separate PR. The whole point of PRs is that if there's an issue the code can be easily reverted in one chunk. if you have all these little changes in there it doesn't do any good.